keras_contrib/applications/densenet.py

```diff
@@ -442,38 +725,58 @@ def __create_dense_net(nb_classes, img_input, include_top, depth=40, nb_dense_bl
         bottleneck: add bottleneck blocks
         reduction: reduction factor of transition blocks. Note : reduction value is inverted to compute compression
         dropout_rate: dropout rate
-        weight_decay: weight decay
+        weight_decay: weight decay rate
+        subsample_initial_block: Set to True to subsample the initial convolution and
```

Comment:
Add something like `Typically True for imagenet pretrained weights, False for cifar pretrained weights`.